src/Utils/RenderReplayError.tsx

```diff
@@ -27,7 +27,7 @@ export function renderReplayError(replayError: CLM.ReplayError): JSX.Element {
                     <div className={OF.FontClassNames.mediumPlus}>
                         <FormattedMessageId id={FM.REPLAYERROR_DESC_ENTITY_EMPTY} />
                         {` "${(replayError as CLM.ReplayErrorEntityEmpty).values.join(", ")}"`}
-                        <HelpIcon tipType={TipType.REPLAYERROR_DESC_ENTITY_EMPTY} customStyle="cl-icon--transparent" />
```

Comment:
Is cl-icon-transparent used any more?  If so remove the class from the .css file and get rid of the "transparentdark" which isn't used.  Thansk!